templates/tfengine/components/devops/terraform.tfvars

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{$missing_admins_group := not (get .admins_group "exists")}}
+admins_group = {
+  {{- if $missing_admins_group}}
+  customer_id  = "{{.admins_group.customer_id}}"
+  {{- end -}}
```

Comment:
all attributes except `id` should be included in this block right? because those fields are only needed when missing_admins_groups is true.

Reply:
You're right, thanks!